docs/contributing/assets/allowed_actions.json

```
@@ -16,6 +16,22 @@
       },
       "comment": ""
     },
+    {
+      "name": "golangci/golangci-lint-action",
+      "versions": [
+        "3cfe3a4abbb849e10058ce4af15d205b6da42804",
+        "v4"
```

Please use the commit hash or the full version (including all minor versions, [current version is 4.0.0](https://github.com/golangci/golangci-lint-action/releases/tag/v4.0.0))
```suggestion
        "v4.0.0"
```